### PR TITLE
Some code to treat Gamma conversions. Unnecessary changes removed

### PR DESCRIPTION
**Pull request by sdifalco**

### CODE REVIEW

**Comment by kutschke** [comment]
@sdifalco Please use the edit feature in the box at the top of this page to provide an overview description of this PR.

**Comment by kutschke** [comment]
@FNALbuild run build test with Mu2e/Production#279

File: Print/src/PrintModule_module.cc
Comment (sdifalco) [inline Print/src/PrintModule_module.cc:1]: all the changes of this should be deleted

**Comment by kutschke** [comment]
@FNALbuild run build test with Mu2e/Production#279

**Comment by kutschke** [comment]
@FNALbuild run build test with Mu2e/Production#279

**Comment by kutschke** [comment]
@sdifalco for future reference, the code author can launch the CI job.  The other people who can are people with write access (or maybe admin access?) to the repo.  So you can launch it yourself on your own PRs.  If you copy and paste a previous command it won't work because of the embedded hyperlink.  So you need to type it out ( or paste into and editor that loses the formatting and then copy and paste from there).

**Review by sophiemiddleton — APPROVED** [review]
this looks fine to me

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (brownd1978) [inline EventGenerator/src/GammaConvGenerator_module.cc:5]: Is this the right author?

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (brownd1978) [inline EventGenerator/src/GammaConvGenerator_module.cc:28]: Is this include used?  Could you check what of these are actually used?

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (brownd1978) [inline EventGenerator/src/GammaConvGenerator_module.cc:1]: Please provide a more accurate description

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (brownd1978) [inline EventGenerator/src/GammaConvGenerator_module.cc:99]: Since this module produces a flat momentum distribution, while the physical distribution isn't perfectly flat, I suggest to rename this module GammaConvFlat

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (brownd1978) [inline EventGenerator/src/GammaConvGenerator_module.cc:133]: IIRC the mass of the e+e- pair follows a roughly exponential distribution, peaking at 2me.   What is the assumption here?  0 mass?  Is there any plan to provide a mass distribution in future?

File: MCDataProducts/inc/GenId.hh
Comment (brownd1978) [inline MCDataProducts/inc/GenId.hh:47]: For art resampling we use ProcessCode to describe the production of stopped particle daughters.   That is also how we identify particles in FindMCPrimary.  I see that you use that below as well, why do you also need this genId?

**Review by brownd1978 — CHANGES_REQUESTED** [review]
I've made some minor suggestions.  In addition, I'd like to make sure that FindMCPrimary can work with these particles.   Please look at how that's configured for (say) conversion electrons in https://github.com/Mu2e/Production/blob/main/JobConfig/primary/CeEndpoint.fcl

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (sdifalco) [reply EventGenerator/src/GammaConvGenerator_module.cc:5]: updated

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (sdifalco) [reply EventGenerator/src/GammaConvGenerator_module.cc:28]: cleaned

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (sdifalco) [reply EventGenerator/src/GammaConvGenerator_module.cc:99]: changed, I am in contact with Michael Mackenzie to prepare a more accurate generator taking SImParticle collection as input

File: EventGenerator/src/GammaConvGenerator_module.cc
Comment (sdifalco) [reply EventGenerator/src/GammaConvGenerator_module.cc:133]: I impose momentum conservation neglecting nucleus recoil and the invariant mass is peaked at 2me I have added slide 9 to doc-db 47028 to show this

File: MCDataProducts/inc/GenId.hh
Comment (sdifalco) [reply MCDataProducts/inc/GenId.hh:47]: I found it useful in the dump of SimParticle collection

**Review by brownd1978 — COMMENTED** [review]
Thanks for making the changes.  I'm confused though that there is now both GammaConvFlat_module and GammaConvGenerator_module, both producing a flat spectrum, was that intentional?

**Comment by brownd1978** [comment]
@FNALbuild run build test with Mu2e/Production#279

**Comment by brownd1978** [comment]
3 whitespace errors were found , as follows.  I don't want to hold up this PR, but please fix them and make a new PR.
CommonMC/src/StoppedParticlesFinder_module.cc:216: trailing whitespace, tab in indent.
+	std::cout <<  "STAGE " << particle.simStage() << " vs " << simStageThreshold_ << " pid=" <<particle.pdgId() << 
CommonMC/src/StoppedParticlesFinder_module.cc:217: tab in indent.
+	  " stopcode=" << particle.stoppingCode().id() << " name=" <<  particle.stoppingCode().name() << std::endl;
CommonMC/src/StoppedParticlesFinder_module.cc:223: tab in indent.
+	   && ( isStopped(particle) || particle.stoppingCode().id()==13 ) ) //13=photon conversion

**Pull request merged**
